src/package-lock.json

```
@@ -1,12 +1,12 @@
 {
-  "name": "trust-web3-provider",
-  "version": "1.0.0",
+  "name": "@trustwallet/trust-web3-provider",
```

since we have `@trustwallet` as a prefix
```suggestion
  "name": "@trustwallet/web3-provider",
```